etna/transforms/statistics.py

```diff
@@ -196,16 +238,14 @@ def _aggregate_window(self, series: pd.Series) -> float:
 class QuantileTransform(WindowStatisticsTransform):
     """QuantileTransform computes quantile value for given window."""
 
-    default_out_postfix = "quantile"
-
     def __init__(
         self,
         in_column: str,
         quantile: float,
         window: int,
+        out_column: Optional[str] = None,
```

Comment:
`out_column` at this position (between `window` and `seasonality`) looks strange because it splits similar parameters to two groups; can we move it to the last position?

Reply:
done